mmv1/third_party/terraform/tests/resource_compute_backend_service_test.go.erb

```diff
@@ -731,6 +731,37 @@ func TestAccComputeBackendService_withCompressionMode(t *testing.T) {
 	})
 }
 
+func TestAccComputeBackendService_trafficDirectorUpdateLbPolicies(t *testing.T) {
```

Comment:
Do you mind adding a similar test for RegionBackendService as well? Exercising the same values is fine- I'd like to ensure that we're calling both the `backendServices` and `regionBackendServices` endpoints.

Reply:
I added this test before quickly realizing that the new field is not applicable to region backend services! I had just included the change in region backend service in api.yaml months ago by mistake. I removed it from there now.

Reply:
Ha, that's what we test these kind of things to find out. Thanks!